.github/workflows/ci.yaml

```diff
@@ -44,7 +43,7 @@ jobs:
       - name: Lint golang code
         uses: golangci/golangci-lint-action@v3
         with:
-          version: ${{ env.GOLANGCI_VERSION }}
+          version: latest
```

Comment:
why not version v1.51.1 is used here?

Reply:
It is convenient to use latest without modifying it every time

Reply:
@fengshunli In file `Makefile`, you will install golangci-lint version v1.51.1, but here latest version is used, so installed version is not used.

Reply:
Makefile has been updated @rambohe-ch